editor/plugins/script_editor_plugin.cpp

```diff
@@ -2330,6 +2277,59 @@ bool ScriptEditor::edit(const RES &p_resource, int p_line, int p_col, bool p_gra
 	return true;
 }
 
+void ScriptEditor::save_current_script() {
```

Comment:
I moved this method to be next to `save_all_scripts()`, since these are logically related and both methods are now public.